packages/gatsby-theme-project-portal/src/layouts/SearchPageLayout.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import { SearchPageLayout } from "./SearchPageLayout"
+import * as ProjectTeamStories from "../components/ProjectTeam.stories"
+
+const meta: Meta<typeof SearchPageLayout> = {
+  component: SearchPageLayout,
+  tags: ["autodocs"],
+}
+
+export default meta
+
+type Story = StoryObj<typeof SearchPageLayout>
+
+export const Primary: Story = {
+  args: {
+    data: {
+      allGeneralPage: {
+        nodes: {
```

Comment:
Do we need the `nodes` here to be an array? I'm getting an undefined error when I run this on the preview https://deploy-preview-653--project-portal-storybook.netlify.app/?path=/docs/gatsby-theme-project-portal-src-layouts-searchpagelayout--docs
I think it's because the nodes isn't an array, and the `foreach` is then undefined.